jenkins/bootstrap.py

```diff
@@ -881,6 +883,8 @@ def bootstrap(args):
         exc_type, exc_value, exc_traceback = sys.exc_info()
         logging.exception('unexpected error')
         success = False
+    if not setup_creds:
+        setup_credentials(call, args.service_account, upload)
```

Comment:
If we cannot setup credentials we cannot upload logs so the rest of the program is pretty pointless